activitysim/skim.py

```diff
@@ -49,3 +50,229 @@ def get(self, orig, dest):
             dest = dest + self.offset
 
         return self.data[orig, dest]
+
+
+class Skims(object):
+
+    def __init__(self):
+        """
```

Comment:
It's kind of inconsistent, but the class docstring doesn't go in the `__init__` method, it goes right after the `class Foo(object):` line. That includes the description of the parameters to `__init__`.